apps/bored/bored.star

```diff
@@ -0,0 +1,204 @@
+"""
+Applet: Bored
+Summary: Things to do when bored
+Description: This app will suggest things you can do alone or with your friends if you are bored.
+Author: Anders Heie
+"""
+
+load("render.star", "render")
+load("schema.star", "schema")
+load("http.star", "http")
+load("encoding/base64.star", "base64")
+load("cache.star", "cache")
+load("random.star", "random")
+
+# Global defines
+BORED_URL = "https://www.boredapi.com/api/activity"
+DEFAULT_COLOR = "#FF00FF"
+DEFAULT_FRIENDS = "random"
+DEFAULT_FONT = "6x13"
+DEFAULT_DIRECTION = "vertical"
+
+def main(config):
+    activity = cache.get("activity")
```

Comment:
Since cache key is the same regardless of the provided params, most users will just see the result for whatever params that happened to be provided when cache was populated.

Reply:
Ah, thanks. My first app, I'll fix it.